### PR TITLE
Edited Instructions for Team Edition Upgrade

### PR DESCRIPTION
**Pull request by DavidLu1997**
Edited instructions for upgrading to 3.0.x. Changes were made for clarity, conciseness, and readability.

TODO: Move Enterprise instructions to ee-install.

@it33 please review

### CODE REVIEW

File: source/administration/upgrade.md [hunk 1]
```diff
@@ -4,48 +4,51 @@ This guide explains how to upgrade your Mattermost deployment across versions an
...
+      1. Run `platform -version` to confirm the current version of your Mattermost server is `v2.2.0`, `v2.1.1`, or `v2.0.0`. If not, please upgrade to [`v2.0.0`](http://docs.mattermost.com/administration/upgrade.html#upgrading-team-edition-prior-to-3-x).
       2. Run `wget https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-linux-amd64.tar.gz` to download the appropriate new version. 
 2. Stop the Mattermost Server
-      1. Consider posting an announcement to active teams about stopping the Mattermost server for an upgrade
-      2. To stop the server run `sudo stop mattermost`
-3. Backup your data
+      1. Consider announcing to active teams that the Mattermost server will be stopped for an upgrade
+      2. Run `sudo stop mattermost`
+3. Back up your data
       1. Back up your `config.json` file, which contains your system configuration. This will be used to restore your current settings. after the new version is installed.
```
Comment (DavidLu1997) [inline source/administration/upgrade.md]: Add exact location of `config.json` if it is not in the root directory.

File: source/administration/upgrade.md [hunk 2]
```diff
@@ -4,48 +4,51 @@ This guide explains how to upgrade your Mattermost deployment across versions an
...
+                  - If your server contains duplicate accounts (multiple accounts with either the same email addresses or the same usernames), see [duplicate accounts](http://docs.mattermost.com/administration/upgrade.html#duplicate-accounts).
 7. Start your server and address any setting changes relevant in the latest version of Mattermost
       1. Run `sudo start mattermost`.
       2. Opening the **System Console** and saving a change will upgrade your `config.json` schema to the latest version using default values for any new settings added. 
 8. Test the system is working by going to the URL of an existing team.
       1. You may need to refresh your Mattermost browser page in order to get the latest updates from the upgrade.
-9. After the Mattermost 3.0 upgrade users with duplicate accounts can follow instructions in the upgrade email they received to login to teams on which the duplicate accounts were created and add their primary account to the team and any private groups that are still actively used. Users can continue to access the direct message history of their duplicate accounts using their updated email addresses.
+
+#### Duplicate Accounts
```
Comment (it33) [inline source/administration/upgrade.md]: @DavidLu1997 not sure if this section is rendering as you intended? 

image

**Comment by DavidLu1997** [comment]
Resolved merge conflicts, reformatted.

**Comment by it33** [comment]
Not clear why this is an improvement. Also, upgrade docs have changed significantly since this PR opened.

**Pull request closed without merging**